.github/dependabot.yml

```diff
@@ -11,23 +11,19 @@ updates:
       # Managed by cisagov/skeleton-generic
       - dependency-name: actions/cache
       - dependency-name: actions/checkout
+      - dependency-name: actions/dependency-review-action
       - dependency-name: actions/setup-go
       - dependency-name: actions/setup-python
+      - dependency-name: cisagov/action-job-preamble
       - dependency-name: cisagov/setup-env-github-action
-      - dependency-name: crazy-max/ghaction-dump-context
       - dependency-name: crazy-max/ghaction-github-labeler
-      - dependency-name: crazy-max/ghaction-github-status
-      - dependency-name: GitHubSecurityLab/actions-permissions
+      - dependency-name: github/codeql-action
       - dependency-name: hashicorp/setup-packer
       - dependency-name: hashicorp/setup-terraform
       - dependency-name: mxschmitt/action-tmate
-      - dependency-name: step-security/harden-runner
       # Managed by cisagov/skeleton-ansible-role
+      - dependency-name: cisagov/action-disable-apparmor
       - dependency-name: docker/setup-buildx-action
-      - dependency-name: docker/setup-qemu-action
-      - dependency-name: github/codeql-action
-      # # Managed by cisagov/skeleton-ansible-role-with-test-user
-      # - dependency-name: aws-actions/configure-aws-credentials
```

Comment:
I don't think this should be removed. Is there something I'm missing?

Reply:
I think you're right @mcdonnnj - I overlooked that.  @jsf9k can confirm tho.

Reply:
Please see commit 95e89bf.